### ISSUE TITLE
Update Collection Setting Data Structure

### ISSUE THREAD

**Issue by Davinkjohnson** [episphere/connect#507]
Use the example nested JSON created by Tony to nest the Collect Setting within the Participants table.

**Comment by anthonypetersen** [comment]
![Screen Shot 2022-04-12 at 9 06 14 AM](https://user-images.githubusercontent.com/18222069/163034731-2f67cec6-6188-44c3-a8ec-4b7e2f8ae04e.png)

**Comment by anthonypetersen** [comment]
@FrogGirl1123 also want to verify... do we want concepts as well to track the times? or are these variables only concerned with locations

**Comment by anthonypetersen** [comment]
![image](https://user-images.githubusercontent.com/18222069/163044129-4102f07e-a603-4317-be88-6467ddf3bab9.png)

**Comment by Davinkjohnson** [comment]
@anthonypetersen we should already have the collection times on the Participants table, right? (Under the check-in time?)

**Comment by anthonypetersen** [comment]
@FrogGirl1123 another thought... when this implementation is done, are the concepts above going to be necessary?

**Comment by anthonypetersen** [comment]
@Davinkjohnson we only track the collection time of baseline blood, urine, and mouthwash on the participants table

**Comment by brotzmanmj** [comment]
A couple of small comments... the CIDs look correct to me but just for clarity, these are Collection Setting (not Collection Location which we do have variables for but they are different than Collection Setting). Tony you're correct that the ones you listed are the ones we wanted in the Participants table though of course Nicole should confirm. About collection times, we do want the collection times of the samples (bloodtime, urinetime, mouthwashtime that Tony has the CIDs for above) in the Participants table. I thought we already had them there but if we don't, then yes please include them. @Davinkjohnson we also have visit check-in time and check-out time in the Participant table but those are different variables than the collection times.

**Comment by anthonypetersen** [comment]
![image](https://user-images.githubusercontent.com/18222069/163192654-df18da12-f5f7-4bea-9ed7-9fc586b95338.png)

**Comment by anthonypetersen** [comment]
@FrogGirl1123 @brotzmanmj above is a more detailed version of what I have in mind.. just to clarify for my sake, nothing above would be considered "default" correct? In my head this would work similarly to the Check-In nesting (331584571) where initially nothing exists and we are only creating and setting CIDs as they are needed. Thoughts?

**Comment by brotzmanmj** [comment]
@anthonypetersen This is going to take more time to consider than I have now on PTO so I suggest holding on this until I return next week. My concern with the structure that's depicted above is with the placement of this: [COLLECTION SETTING CID] (New) so high up in the structure. That doesn't seem to be right to me.

**Comment by brotzmanmj** [comment]
@anthonypetersen  Your understanding on this is correct:  "My understanding is that the Setting would be equal to the Setting of the Collection where a participant actually had a certain Specimen Type collected. So say a participant never actually gave baseline blood, then they would never have the Blood Setting CID set for Baseline."

**Comment by anthonypetersen** [comment]
@brotzmanmj sounds good

**Comment by FrogGirl1123** [comment]
@brotzmanmj Yes, this can wait until next week.  For clarification, the Collection Setting CID is a the same level as the Payment 
Round CID and the Visit CID, it allows us to harmonize/reuse the CIDs for Baseline, Follow-up 1, Follow-up 2, etc. because they are nested under the specific use.

**Comment by brotzmanmj** [comment]
@FrogGirl1123 Thanks very much, feel free to schedule whatever time is convenient for you to meet next week. One question...are you saying you would have parallel structures for the biospecimens in the Participants Table, the one shown above nested under the Collection Setting CID as well as the existing one that's nested under the Visit CID? Would that generate redundant data for the same sample collections? Is there a reason not have the existing structure for example Visit = baseline, then under that bloodcollected, bloodtime, blood setting, urinecollected, urinetime, urinesetting, mouthwashcollected, mouthwashtime, mouthwashsetting? Maybe that's not possible but that was kinda where I assumed it would go.

**Comment by anthonypetersen** [comment]
@brotzmanmj are you suggesting we don't create a new CID for collection setting, and instead put all of this information under 331584571?

**Comment by FrogGirl1123** [comment]
That's a possibility, but let's talk out the pros and cons next week.  I'll try to find us a meeting for Monday or Tuesday.
